### PR TITLE
Planning: exported open space zig-zag trajectory time latency.

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/planning/proto/planning_internal.proto [hunk 1]
```diff
@@ -15,9 +15,7 @@ import "modules/planning/proto/sl_boundary.proto";
 import "modules/planning/proto/decision.proto";
 import "modules/planning/proto/planning_config.proto";
 
-message Debug {
-  optional PlanningData planning_data = 2;
-}
+message Debug { optional PlanningData planning_data = 2; }
```
Comment (jmtao) [inline modules/planning/proto/planning_internal.proto]: new line

File: modules/planning/proto/planning_internal.proto [hunk 2]
```diff
@@ -113,18 +111,16 @@ message ReferenceLineDebug {
...
   optional double dkappa_rms = 12;
   optional double kappa_max_abs = 13;
   optional double dkappa_max_abs = 14;
   optional double average_offset = 15;
 }
 
-message SampleLayerDebug {
-  repeated apollo.common.SLPoint sl_point = 1;
-}
+message SampleLayerDebug { repeated apollo.common.SLPoint sl_point = 1; }
```
Comment (jmtao) [inline modules/planning/proto/planning_internal.proto]: new line

File: modules/planning/proto/planning_internal.proto [hunk 3]
```diff
@@ -137,9 +133,7 @@ message ScenarioDebug {
   optional string msg = 3;
 }
 
-message Trajectories {
-  repeated apollo.common.Trajectory trajectory = 1;
-}
+message Trajectories { repeated apollo.common.Trajectory trajectory = 1; }
```
Comment (jmtao) [inline modules/planning/proto/planning_internal.proto]: new line

File: modules/planning/proto/planning_internal.proto [hunk 4]
```diff
@@ -230,9 +225,7 @@ message LatticeStTraining {
   optional double t_resolution = 7;
 }
 
-message CostComponents {
-  repeated double cost_component = 1;
```
Comment (Capri2014) [inline modules/planning/proto/planning_internal.proto]: same here

File: modules/planning/proto/planning_internal.proto [hunk 5]
```diff
@@ -159,6 +153,7 @@ message OpenSpaceDebug {
   optional apollo.planning_internal.Trajectories fallback_trajectory = 15;
   optional apollo.common.TrajectoryPoint trajectory_stitching_point = 16;
   optional apollo.common.TrajectoryPoint future_collision_point = 17;
+  optional double time_latency = 18; // ms
```
Comment (Capri2014) [inline modules/planning/proto/planning_internal.proto]: please use more trial name, like `time_latency_ms`?

Comment (sjiang2018) [reply modules/planning/proto/planning_internal.proto]: In that case, if we decide to use sec as unit in future, we will have to rename this field.

**Comment by sjiang2018** [comment]
@Capri2014 @jmtao The protolint (https://github.com/yoheimuta/protolint/blob/master/README.md) changes certain statements to one-line when the total length is less than 80 characters. If we are not following this lint version, Is there any other proto lint I can refer to?

File: modules/planning/proto/planning_internal.proto (hunk 3 above)
Comment (sjiang2018) [reply modules/planning/proto/planning_internal.proto]: Fixed. Thank you.

File: modules/planning/proto/planning_internal.proto (hunk 2 above)
Comment (sjiang2018) [reply modules/planning/proto/planning_internal.proto]: Fixed. Thank you.

File: modules/planning/proto/planning_internal.proto (hunk 1 above)
Comment (sjiang2018) [reply modules/planning/proto/planning_internal.proto]: Fixed. Thank you.

File: modules/planning/proto/planning_internal.proto (hunk 4 above)
Comment (sjiang2018) [reply modules/planning/proto/planning_internal.proto]: Fixed. Thank you.

**Comment by sjiang2018** [comment]
> @Capri2014 @jmtao The protolint (https://github.com/yoheimuta/protolint/blob/master/README.md) changes certain statements to one-line when the total length is less than 80 characters. If we are not following this lint version, Is there any other proto lint I can refer to?


NVM, issue resolved. Thank you.

**Pull request merged**